docs/tutorials/development-setup.mdx

```diff
@@ -24,6 +24,7 @@ import TabItem from '@theme/TabItem';
   ```shell
   cd fxa
   yarn install
+  yarn l10n:clone
```

Comment:
Q: Do we need to <kbd>yarn l10n:clone</kbd> before or _after_ running <kbd>yarn install</kbd>?
I just did a fresh install w/
- `gh repo clone mozilla/fxa`
- `cd fxa`
- `nvm use`
- `yarn install`

But then I see this near the bottom of my yarn logs:
```sh
➤ YN0009: │ fxa-payments-server@workspace:packages/fxa-payments-server couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/54/3n1x5kvx73l6xmg1dj0cgx5w0000gn/T/xfs-ce306213/build.log)
```

Where the build.log says:
```sh
# This file contains the result of Yarn building a package (fxa-payments-server@workspace:packages/fxa-payments-server)
# Script name: postinstall

[l10n/prime]: No external/l10n folder exists! Run yarn l10n:clone script first.
```

But then if I delete my (1.6 GB?) node_modules on the fresh install and run <kbd>yarn l10n:clone</kbd> first, I seem to get this error:
```sh
yarn l10n:clone
Usage Error: Couldn't find the node_modules state file - running an install might help (findPackageLocation)

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...
```

If I do an install, and let it fail at the end, THEN do a `yarn l10n:clone` it succeeds, then I can do a `yarn install` again and it seems to complete successfully.

Reply:
Hmm, yeah there is a bit of a chicken egg issue here due to the way we use post install. In general I don't think we should rely on post install so much, since it has some [side effects](https://yarnpkg.com/advanced/lifecycle-scripts#a-note-about-postinstall).

For now I will change it so this works, but hopefully in the near future we can run yarn install, then yarn clone:l10n without error.

I've adjusted the instructions accordingly.